lib/Form/Listing/FormListingRepository.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace MailPoet\Form\Listing;
+
+use MailPoet\Entities\FormEntity;
+use MailPoet\Listing\ListingDefinition;
+use MailPoet\Listing\ListingRepository;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoetVendor\Doctrine\ORM\QueryBuilder;
+
+class FormListingRepository extends ListingRepository {
+  public function getGroups(ListingDefinition $definition): array {
+    $queryBuilder = clone $this->queryBuilder;
+    $this->applyFromClause($queryBuilder);
+    $this->applyParameters($queryBuilder, $definition->getParameters());
+
+    // total count
+    $countQueryBuilder = clone $queryBuilder;
+    $countQueryBuilder->select('COUNT(f) AS formCount');
+    $countQueryBuilder->andWhere('f.deletedAt IS NULL');
+    $totalCount = (int)$countQueryBuilder->getQuery()->getSingleScalarResult();
+
+    // trashed count
+    $trashedCountQueryBuilder = clone $queryBuilder;
+    $trashedCountQueryBuilder->select('COUNT(f) AS formCount');
+    $trashedCountQueryBuilder->andWhere('f.deletedAt IS NOT NULL');
+    $trashedCount = (int)$trashedCountQueryBuilder->getQuery()->getSingleScalarResult();
+
+    return [
+      [
+        'name' => 'all',
+        'label' => WPFunctions::get()->__('All', 'mailpoet'),
```

Comment:
We don't have to use WPFunctions for translation functions like __, _x, etc. It is an exception. But there is no need to fix that. Just so that you know.

Reply:
Thanks for the information. I pushed a commit fixing this anyway for the sake of consistency and to help me remember about this next time.

Would it make sense to remove \MailPoet\WP\Functions::__() and other i18n functions and replace all the usages?

Reply:
I think it is something we should do so that it is used consistently. It is probably not a high priority though. I remove it when I see in the code I'm touching.